packages/react/src/components/Skeleton/Rectangle/Rectangle.tsx

```diff
@@ -0,0 +1,37 @@
+import React, { type HTMLAttributes } from 'react';
+import cn from 'classnames';
+
+import { useSynchronizedAnimation } from '../../../hooks';
+import classes from '../Skeleton.module.css';
+
+export type RectangleProps = {
+  /** The width of the component */
+  width?: string | number;
+  /** The height of the component */
+  height?: string | number;
+} & HTMLAttributes<HTMLDivElement>;
+
+/**  Skeleton component used for indicating loading elements of rectangle shape */
+export const Rectangle = ({
+  width,
+  height,
+  className,
+  children,
+  ...rest
+}: RectangleProps) => {
+  const ref = useSynchronizedAnimation<HTMLDivElement>('opacity-fade');
+
+  return (
+    <div
+      {...rest}
+      ref={ref}
+      className={cn(classes.skeleton, classes.rectangle, className, {
+        [classes.hasChildren]: Boolean(children),
+      })}
```

Comment:
Their `className` should ideally come last, so we make sure they have the most specificity